### ISSUE TITLE
Not joining raids

### ISSUE THREAD

**Issue by Sk3tchy-mctw1dd** [bttv-android/bttv#658]
### Existing Issues

- [X] I have searched for existing issues, this is not a duplicate.

### Unmodded Twitch

- [X] I have checked and this issue only occurs using bttv-android and not with the unmodded Twitch App

### What happened?

It seem that whenever a streamer I am watching finishes and then goes on to raid another, unless I have twitch open to that channel in another instance (i.e. twitch mobile or website), I get no raid notification and don't join the raid. 

### bttv-android version

V0.40.4

### android version

14

### device

Google pixel 6

**Comment by LunarRoze** [comment]
having the same issue rn, raids just won't work on bttv

**Comment by Sk3tchy-mctw1dd** [comment]
Just happened, had my laptop open to the same stream (I had walked away from it for a bit), and even though the raid message came up due to the laptop being open to it, the stream just ended. I did notice that the raided stream went straight to ads for me from the raid.

**Comment by jraug44** [comment]
It seems like the app just breaks randomly because sometimes raids don't work or when there's a hype train the time is stuck at 0:01 and it doesn't go to the next level, also sometimes redeem and gifted sub notifications don't show up, and I have to restart the app to fix it

**Comment by jobbio** [comment]
I've found that I have to fully close the app and re-open it before the raid happens for it to consistently send me through the raid. If I've been watching the stream for a while and close the screen a number of times, it is unlikely that the raid will work for me.

**Comment by Sk3tchy-mctw1dd** [comment]
Yes. I just updated to the new version of the apk this past weekend, and I had it running all evening while I was camping, and it didn't notify or pull me along with the raid, just showed that the broadcast had ended
